instagram_private_api/endpoints/accounts.py

```diff
@@ -75,32 +75,38 @@ def current_user(self):
             ClientCompatPatch.user(res['user'], drop_incompat_keys=self.drop_incompat_keys)
         return res
 
-    def edit_profile(self, first_name, biography, external_url, email, phone_number, gender):
+    def edit_profile(self, username=None, first_name=None, biography=None,
+                     external_url=None, email=None, phone_number=None, gender=None):
         """
         Edit profile
-
+        
+        :param username:
         :param first_name:
         :param biography:
         :param external_url:
-        :param email: Required.
+        :param email:
         :param phone_number:
         :param gender: male: 1, female: 2, unspecified: 3
         :return:
         """
-        if int(gender) not in [1, 2, 3]:
-            raise ValueError('Invalid gender: {0:d}'.format(int(gender)))
-        if not email:
-            raise ValueError('Email is required.')
-
+        current_user = self.current_user()['user']
+    
+        if gender is None:
```

Comment:
A None value will evaluate as false, so this statement can be simplified to: ``if not gender:``